cmd/kubeadm/app/features/features.go

```diff
@@ -33,7 +33,7 @@ const (
 	PublicKeysECDSA = "PublicKeysECDSA"
 	// RootlessControlPlane is expected to be in alpha in v1.22
 	RootlessControlPlane = "RootlessControlPlane"
-	// EtcdLearnerMode is expected to be in alpha in v1.26
+	// EtcdLearnerMode is expected to be in alpha in v1.27
```

Comment:
I'd prefer to squash this commit, it is not necessary a separate commit.

Reply:
This may help if you review the change.
- I will squash them once all comments are addressed.